app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -49,6 +49,8 @@ def create
     auth_token = UserService::API::AuthTokens.create_login_token(user[:id])
     url = URLUtils.append_query_param(marketplace[:url], "auth", auth_token[:token])
 
+    ab_test_onboarding(community_id: marketplace[:id], alternative_flags: [:onboarding_new, :onboarding_old])
```

Comment:
This needs a global conditional ```if(featured_enabled(:onboarding_ab_test))``` or we don't merge this branch to master before it's ready.